.github/workflows/CI.yml

```diff
@@ -12,6 +12,14 @@ on:
       - 'master'
       - 'main'
       - 'release-*'
+  # Allow users to manual run the CI pipeline for a specific branch, this may
+  # be needed for the integration tests as they need the generated artifacts / images
+  workflow_dispatch: # run integration tests only when triggered manually
+    inputs:
+      branch:
+        description: 'Branch (e.g., main, release-x.y.z) for which artifacts should be build'
+        required: true
+        default: 'main'
```

Comment:
I don't think this has any affect in the CI workflow - branch is selected in the UI directly
![image](https://user-images.githubusercontent.com/56065213/165081025-fd21d2b3-2f5d-4e4b-bb6f-8bebc46aa55e.png)